hack/manifests/testing/parodos-patch.yaml

```diff
@@ -20,10 +20,12 @@ data:
   DATASOURCE_PASSWORD: "parodos"
   SPRING_PROFILES_ACTIVE: "dev"
   NOTIFICATION_SERVER_ADDRESS: "notification-service.default.svc.cluster.local"
+  WORKFLOW_SERVICE_PATH: "/workflow-service"
 ---
 apiVersion: v1
 kind: ConfigMap
 metadata:
   name: notification-service-config
 data:
   SPRING_PROFILES_ACTIVE: "local"
+  NOTIFICATION_SERVICE_PATH: "/notification-service"
```

Comment:
Where is `NOTIFICATION_SERVICE_PATH` being used?

Reply:
[here](https://github.com/parodos-dev/parodos/blob/39b4eded68caa276a5b916ce2c69434413898b44/.github/workflows/test.yml#L199) and [here](https://github.com/parodos-dev/parodos/blob/39b4eded68caa276a5b916ce2c69434413898b44/sdk-utils/src/main/java/com/redhat/parodos/sdkutils/NotificationServiceSdkUtils.java#L60)